youtube_dl/utils.py

```diff
@@ -2371,15 +2371,24 @@ def set_alpn_protocols(ctx):
         return YoutubeDLHTTPSHandler(params, context=context, **kwargs)
 
 
-def bug_reports_message():
+def bug_reports_message(before=';'):
     if ytdl_is_updateable():
         update_cmd = 'type  youtube-dl -U  to update'
     else:
-        update_cmd = 'see  https://yt-dl.org/update  on how to update'
-    msg = '; please report this issue on https://yt-dl.org/bug .'
-    msg += ' Make sure you are using the latest version; %s.' % update_cmd
-    msg += ' Be sure to call youtube-dl with the --verbose flag and include its complete output.'
-    return msg
+        update_cmd = 'see  https://github.com/ytdl-org/youtube-dl/#user-content-installation  on how to update'
+
+    msg = (
+        ' please report this issue on https://github.com/ytdl-org/youtube-dl/issues ,'
```

Comment:
Remove initial space that is added in l.2391 if needed.
```suggestion
        'please report this issue on https://github.com/ytdl-org/youtube-dl/issues ,'
```